src/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -198,7 +183,7 @@ public static void IsZipSameAsDir(Stream archiveFile, String directory, ZipArchi
                         if (entry == null) //entry not found
                         {
                             string entryNameOtherSlash = FlipSlashes(entryName);
-                            Boolean isEmtpy = !allFilesInDir.Any(
+                            Boolean isEmtpy = !files.Any(
```

Comment:
There are quite a few of these throughout System.IO.Compression. I'll fix this one but I think I'll save the rest for a PR dedicated to that purpose so the diff doesn't get too complicated.